MMM-pages.js

```diff
@@ -178,14 +176,14 @@ Module.register('MMM-pages', {
    * elements.
    */
   updatePages: function () {
-    // Update iff there's at least one page.
+    // Update if there's at least one page.
```

Comment:
This isn't a typo, it's a math shorthand for ["if and only if"](https://en.wikipedia.org/wiki/If_and_only_if).

I think someone else tried to correct it in #48 as well a while back too, so maybe it's just easier if we change it.

I'll let you choose whether to change it or leave it as it was -- I'm pretty indifferent to this.